Classes/Issues/IssuesViewController.swift

```diff
@@ -239,6 +239,12 @@ IssueManagingContextControllerDelegate {
         )
     }
 
+    func themeDidChange(_ theme: Theme) {
+        view.backgroundColor = Styles.Colors.background
```

Comment:
In my opinion, this code looks heavy
I've used this library for setup themes in my recent project: https://github.com/jiecao-fm/SwiftTheme
Maybe we will take some ideas from there?
It will be more readable, for example, we can simply write something like this:
```swift
view.themeBackgroundColor = Theme.Colors.background
```

Reply:
What does the `themeBackgroundColor` property do?
If your talking about where the color was derived from, I was following the preexisting constants that are in the `Styles`

Reply:
`themeBackgroundColor` sets the background color of a view 🙂
I mean in your implementation we have to listen to the theme change in each file and write code for this. With SwiftTheme we won't need this

Reply:
@ismetanin The problem is even with `SwiftTheme` we would have to listen for theme changes, as GitHawk uses a lot of attributedStrings and so some redrawing of those might be required on theme change